Model/AbstractFragment.php

```diff
@@ -49,18 +49,23 @@
     protected $position;
 
     /**
-     * @var int
+     * @var string
+     */
+    protected $backofficeTitle;
```

Comment:
can't set it private, all other properties are protected, and we used them (on fixtures for exemple)